istioctl/cmd/revision.go

```diff
@@ -891,7 +884,7 @@ func printPodTable(w io.Writer, pods []*tag.PodFilteredInfo) error {
 	return podTableW.Flush()
 }
 
-func getEnabledComponents(iop *iopv1alpha1.IstioOperator) ([]string, error) {
+func getEnabledUserFacingComponents(iop *iopv1alpha1.IstioOperator) ([]string, error) {
```

Comment:
Looks like we have a function in https://github.com/istio/istio/blob/master/operator/pkg/translate/translate_common.go#L94, which is quite the same, can we reuse the logic?